sct.py

```diff
@@ -767,6 +768,12 @@ def create_test_release_jobs(branch, username, password, sct_branch, sct_repo):
             for jenkins_file in glob.glob(f'{server.base_sct_dir}/jenkins-pipelines/admission_control_overload*'):
                 server.create_pipeline_job(jenkins_file, group_name)
 
+    server.create_directory(name='artifacts-offline-install', display_name='SCT Artifacts Offline Install Tests')
+    for jenkins_file in glob.glob(f'{server.base_sct_dir}/jenkins-pipelines/nonroot-offline-install/*.jenkinsfile'):
```

Comment:
Nonroot offline install of centos7 isn't supported, there is no artifacts-centos7.jenkinsfile in nonroot-offline-install/ directory.

Root offline install job of centos7 won't be created in this way.

Reply:
BTW, the 'nonroot_offline_install' parameter value is different from the artifact pipeline in top directory.

We discussed to not reuse the same pipeline by passing different values, so we created the nonroot offline install, it's not guaranteed that the trigger can pass correct parameter (eg: nonroot_offline_install), the trigger may rely on the default value in pipeline.  So we can use nonroot_offline_install pipeline for root offline install jobs.

Reply:
@amoskong, I'm not sure I understand anything you are saying here, feel free to take this commit, and align it.
I was looking on what's on master and trying to do my best to mimic it. (if we have excess jobs, we can disable/delete them)

Reply:
I just manually add root offline install job for centos7:

- https://jenkins.scylladb.com/job/scylla-4.4/job/artifacts-offline-install/job/artifacts-centos7-test/